cloud-inquisitor/notification/notifier_test.go

```diff
@@ -0,0 +1,130 @@
+package notification
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/gobuffalo/packr"
+)
+
+func TestMessageGeneration(t *testing.T) {
+	subject := "subject"
+	to := []string{"toemail"}
+	cc := []string{"ccemail"}
+
+	msg, err := NewHijackNotficationMessage(subject, to, cc, HijackNotificationContent{})
+	if err != nil {
+		t.Fatal(err.Error())
+	}
+
+	if !reflect.DeepEqual(to, msg.To) {
+		t.Fatal("to emails dont match")
+	}
+
+	if !reflect.DeepEqual(cc, msg.CC) {
+		t.Fatal("cc emails dont match")
+	}
+}
+
+func TestMessageNoChain(t *testing.T) {
+	content := HijackNotificationContent{
+		PrimaryResource:     "test.bucket.com",
+		PrimaryResourceType: "S3 Bucket",
+		PrimaryAccountId:    "12345678",
```

Comment:
Even though this is a test, should you not use  12-digit integer for AccountId? It might affect formatting?